CHANGELOG.md

```diff
@@ -31,8 +31,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * `jj status` now supports filtering by paths. For example, `jj status .` will
   only list changed files that are descendants of the current directory.
 
-* File path arguments now support [file pattern
-  syntax](docs/filesets.md#file-patterns).
+* A new config option `ui.allow-filesets` has been added to enable ["fileset"
+  expressions](docs/filesets.md).
```

Comment:
Just to set expectations, I think it would be nice to say this is experimental right in the release note, and we will likely enable it by default in the future:

```suggestion
* A new config option `ui.allow-filesets` has been added to enable ["fileset"
  expressions](docs/filesets.md). Note that filesets are currently experimental, but will
  be enabled by default in a future release.
```

Reply:
Updated, thanks!